pkg/operator/controller/garden/garden/reconciler.go

```diff
@@ -544,3 +545,16 @@ func getValidVolumeSize(volume *operatorv1alpha1.Volume, size string) string {
 
 	return size
 }
+
+func valiEnabled(networking operatorv1alpha1.RuntimeNetworking) (bool, error) {
+	for _, cidr := range networking.Pods {
+		if _, ipNet, err := net.ParseCIDR(cidr); err != nil {
+			return false, fmt.Errorf("failed parsing %q as CIDR: %w", networking.Pods, err)
+		} else if ipNet.IP.To4() == nil {
+			// If To4() returns nil, it's IPv6
+			return false, nil
+		}
+	}
+
+	return true, nil
+}
```

Comment:
Maybe we need to improve this logic now or later as it would also disable `vali` in a dual-stack setup. Then again, we currently have no fully working dual-stack setup for local development...